src/powerflow_types.jl

```diff
@@ -8,21 +8,46 @@ struct ACPowerFlow{ACSolver <: ACPowerFlowSolverType} <: PowerFlowEvaluationMode
     check_reactive_power_limits::Bool
     exporter::Union{Nothing, PowerFlowEvaluationModel}
     calculate_loss_factors::Bool
+    generator_slack_participation_factors::Union{
+        Nothing,
+        Dict{String, Float64},
+        Vector{Dict{String, Float64}},
```

Comment:
You might decide to change the typing in `PowerFlowData` to just `Union{Nothing, Vector{Dict{String, Float64}}` but keep the `Dict{String, Float64}` option here. That seems like a fine user interface.